### ISSUE TITLE
issue installing library with GHC 8.10.5

### ISSUE THREAD

**Issue by peterbecich** [xmonad/xmonad-extras#31]
```
% cabal install --lib 
Resolving dependencies...
cabal: Could not resolve dependencies:
[__0] trying: ghc-8.10.5/installed-8.10.5 (user goal)
[__1] next goal: process (user goal)
[__1] rejecting: process-1.6.11.0 (conflict: ghc =>
process==1.6.9.0/installed-1.6.9.0)
```
Thank you

**Comment by psibi** [comment]
I'm not well versed with the way to install via `cabal`, but you could try installing it via stack. It's also available in the latest lts: https://www.stackage.org/lts-18.0/package/xmonad-extras-0.15.3

**Comment by peterbecich** [comment]
Issue resolved with GHC 8.10.7, thank you
